iulianIachimov/src/main/java/dataCollectionTask/Models/AbstractOrder.java

```diff
@@ -0,0 +1,96 @@
+package dataCollectionTask.Models;
```

Comment:
According to Java code convention packages should always be written in all-lowercase.
See https://www.oracle.com/technetwork/java/codeconventions-135099.html

Reply:
That was changed. However, it caused the previous files to be deleted and new files are committed now.